src/node.cc

```diff
@@ -848,6 +849,26 @@ int InitializeNodeWithArgs(std::vector<std::string>* argv,
   if (per_process::cli_options->icu_data_dir.empty())
     credentials::SafeGetenv("NODE_ICU_DATA",
                             &per_process::cli_options->icu_data_dir);
+
+  #ifdef NODE_ICU_DEFAULT_DATA_DIR
```

Comment:
```suggestion
#ifdef NODE_ICU_DEFAULT_DATA_DIR
```
(don't indent preprocessor directives)